packages/strapi-generate-api/templates/bookshelf/service.template

```diff
@@ -21,7 +21,19 @@ module.exports = {
    */
 
   fetchAll: (params) => {
-    return <%= globalID %>.forge(params).query(params).fetchAll({
+    params = strapi.utils.models.convertParams('<%= globalID.toLowerCase() %>', params);
```

Comment:
We should create another variable and not override the `params` which refer to `ctx.query` because we are losing the initial value.